Description: Switch to autohinting to work around blurry rendering with CFF
Origin: https://bugzilla.redhat.com/show_bug.cgi?id=1062903#c2
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=798805

--- a/fontconfig/31-cantarell.conf
+++ b/fontconfig/31-cantarell.conf
@@ -15,4 +15,13 @@
       <string>Cantarell</string>
     </edit>
   </match>
+  <!-- Switch to autohinting to work around blurry rendering with CFF -->
+  <match target="font">
+    <test name="family">
+      <string>Cantarell</string>
+    </test>
+    <edit name="autohint" mode="assign">
+      <bool>true</bool>
+    </edit>
+  </match>
 </fontconfig>
